{
  "Identifier": "GHSA-pp84-v3mw-gg4w",
  "PackageSlug": "pypi/taipy",
  "Title": "Taipy 3.1.1 affected by CVEs on flask-core and pymongo",
  "Description": "CVEs on 3.1.1\n\nFixed on patch versions: \u003e=3.1.2\nand on major releases: \u003e=4.0.0",
  "Date": "2024-10-09",
  "Pubdate": "2024-08-27",
  "AffectedRange": "\u003c4.0.0",
  "FixedVersions": [
    "4.0.0"
  ],
  "AffectedVersions": "All versions before 4.0.0",
  "NotImpacted": "All versions starting from 4.0.0",
  "Solution": "Upgrade to version 4.0.0 or above.",
  "Urls": [
    "https://github.com/advisories/GHSA-pp84-v3mw-gg4w",
    "https://github.com/Avaiga/taipy/security/advisories/GHSA-pp84-v3mw-gg4w",
    "https://github.com/Avaiga/taipy"
  ],
  "CvssV2": "",
  "CvssV3": "",
  "UUID": "c5b7062b-f9e9-4f87-a4c5-cf0965895e3d"
}